### ISSUE TITLE
example with RouterLInk doesn't get the error

### ISSUE THREAD

**Issue by ldavisCEI** [ldavisCEI/vuetify-list-item-to-prop-issue#1]
@KaelWD was 3142f01f56f9794c4688f9af272fdfd5615a3d25 what you had in [mind](https://github.com/vuetifyjs/vuetify/issues/20387#issuecomment-2308651389)?

I don't get the error doing it this way.

Is it still an issue with vue router?

**Comment by KaelWD** [comment]
No, try using a MyLink component that has

```
const link = useLink({ to: '/helloworld' })

<a :href="link.href">
```

**Comment by ldavisCEI** [comment]
Thanks @KaelWD. I tried your MyLink component idea. While working on it, I updated to latest to see if the issue was still happening. Turns out that the issue is fixed by updating to Vue 3.5.6.

Thanks for your help!
